metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -352,7 +352,7 @@ object Messages {
 
   object BloopJvmPropertiesChange {
     def reconnect: MessageActionItem =
-      new MessageActionItem("Restart Bloop")
+      new MessageActionItem("Apply and restart Bloop")
```

Comment:
I reverted back to asking the user because otherwise when multiple metals servers were running this would result in race conditions